src/librustc_resolve/late.rs

```diff
@@ -1135,6 +1108,15 @@ impl<'a, 'b> LateResolutionVisitor<'a, '_> {
         }
     }
 
+    fn resolve_params(&mut self, params: &[Param]) {
+        let mut bindings = smallvec![(false, <_>::default())];
```

Comment:
```suggestion
        let mut bindings = smallvec![(false, Default::default())];
```
(here and in few other places)